### ISSUE TITLE
MVP: Update Vacancy Tab Filter Categories

### ISSUE THREAD

**Issue by silasck** [codeforkansascity/Neighborhood-Dashboard#63]
![drop downs](https://cloud.githubusercontent.com/assets/18126988/17418824/12563a92-5a60-11e6-9bd3-cfe6d18fe0dc.png)

![vacancy tab](https://cloud.githubusercontent.com/assets/18126988/17418834/1fd120f6-5a60-11e6-8eef-73bf6ec53da3.png)

Hey, I know I posted these in Slack, but I wanted to repost them in Github as well. These are the mock ups for the Vacancy tab.

**Comment by JakeLaCombe** [comment]
I don't think more is a good word to use here now that I look at it again. That makes it seem like there is more description available to use as opposed to illustrating that filters are available.

Given that these are more like popups to give a detailed description, I propose instead just to have a button say close on the popup container.

**Comment by JakeLaCombe** [comment]
I looked at it again, and I couldn't find a good use of the popover vs putting a help tooltip on there. When I try it with the popup and an x button, the usability still feels off. From looking at the mockups, you want to provide the user with information that can help them better understand the data they are looking at. Thus, a help tooltip feels like the most appropriate solution here.

I have implemented one here. https://github.com/codeforkansascity/Neighborhood-Dashboard/commit/e48d800061193dde8b4a520f1f77248319f1b40a
